plugins/nanospell/plugin.js

```diff
@@ -190,10 +190,69 @@
 			// reached the end of block,
 			// so just return what we've walked
 			// of the current word.
-			if (word) return { word: word, range: wordRange };
+			if (word) return {word: word, range: wordRange};
 		}
 	};
 
+	function SuggestionsStorage() {
+		this.enabled = false;
+
```

Comment:
I think it would be helpful to have some commenting in this section explaining the various possible scenarios (such as the exception due to localStorage access being caused by a security policy)